bmeg_app/views/app_lit.py

```diff
@@ -0,0 +1,193 @@
+from ..app import app
+from ..db import G
+from ..components import lit, basic_plots as bp
+from .. import appLayout as ly
+import pandas as pd
+import gripql
+import dash
+import dash_table
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output, State
+from plotly.subplots import make_subplots
+import dash_bootstrap_components as dbc
+import json
+
+######################
+# Prep
+######################
+# Main color scheme
+main_colors= ly.main_colors
+styles=ly.styles
+
+# Grab baseDF for Page (multiple genes, multi drugs)
+# baseDF = lit.get_baseDF() # commented out for ONLY dev to speed up loading
+# baseDF.to_csv('bmeg_app/source/basedf.tsv',sep='\t',index=False) # commented out for ONLY dev to speed up loading
+baseDF=pd.read_csv('bmeg_app/source/basedf.tsv',sep='\t') # TEMP TODO change to cached
```

Comment:
Unclear how this file is created/what it is used for.

Reply:
Several functions are pointed to this file. Used as input. Query on line 24 has long runtime and so app has long start up time. Need to cache output of `lit.get_baseDF()`